plugins/myfitnesspal.py

```diff
@@ -2,21 +2,33 @@
 import math
 import requests
 from bs4 import BeautifulSoup
+from dateutil import parser
 
 from cloudbot import hook
 
-scrape_url = "http://www.myfitnesspal.com/food/diary/{}"
+scrape_url = "http://www.myfitnesspal.com/food/diary/{}?date={}"
```

Comment:
It would probably be a good idea to use the `parameters` keyword to `requests.get()` to pass query parameters rather than formatting them in to the URL. For example: 
```python
request = requests.get(scrape_url.format(user), parameters={'date': date})
```
which ensures that the parameter is properly escaped.